x/auth/ante/sigverify_test.go

```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
+	"github.com/golang/mock/gomock"
```

Comment:
It requires the updates of `core/testing` module.

So the golang/mock is retained here